kyuubi-ctl/src/main/scala/org/apache/kyuubi/ctl/ControlCliArguments.scala

```diff
@@ -48,48 +54,88 @@ class ControlCliArguments(args: Seq[String], env: Map[String, String] = sys.env)
         result match {
           case Some(arguments) =>
             command = getCommand(arguments)
-            command.preProcess()
-            cliArgs = command.cliArgs
+            command.validate()
```

Comment:
here command might be null, it is better to provide some prompt instead of NPE

Reply:
when command in not valid, the parser will execute the runEffects logic.
ctl will termindate and throw prompt like "Unknown argument xxx".
<img width="667" alt="image" src="https://user-images.githubusercontent.com/21362040/173240983-4bfc5079-e29b-43a0-b92b-e1dc25e4aa44.png">

when getCommand is executed, the command should be valid.

Reply:
got it